spec.html

```diff
@@ -11472,6 +11472,20 @@ <h1>Realms</h1>
             <emu-note>Once a Parse Node becomes unreachable, the corresponding [[Array]] is also unreachable, and it would be unobservable if an implementation removed the pair from the [[TemplateMap]] list.</emu-note>
           </td>
         </tr>
+        <tr>
+          <td>
+            [[LoadedModules]]
+          </td>
+          <td>
+            a List of Records with fields [[Specifier]] (a String) and [[Module]] (a Module Record)
+          </td>
+          <td>
+            A map from the specifier strings imported by this realm to the resolved Module Record. The list does not contain two different Records with the same [[Specifier]].
```

Comment:
```suggestion
            <p>A map from the specifier strings imported by this realm to the resolved Module Record. The list does not contain two different Records with the same [[Specifier]].</p>
```
When you've got a block-level element (`<emu-note>`), I think it makes more sense for its 'sibling' content to be block-level as well (as in the cell above for `[[TemplateMap]]`).